manifests/mirror/service.pp

```diff
@@ -8,14 +8,16 @@
 # It manages the kafka-mirror service
 #
 class kafka::mirror::service(
-  $consumer_config       = $kafka::params::consumer_config,
-  $producer_config       = $kafka::params::producer_config,
-  $num_streams           = $kafka::mirror::num_streams,
-  $num_producers         = $kafka::mirror::num_producers,
-  $abort_on_send_failure = $kafka::mirror::abort_on_send_failure,
-  $whitelist             = $kafka::mirror::whitelist,
-  $blacklist             = $kafka::mirror::blacklist,
-  $max_heap              = $kafka::mirror::max_heap
+  $service_ensure             = $kafka::mirror::service_ensure,
```

Comment:
I think the introduction to of $service_ensure isn't related to this bug? Can you move it to a separate commit?